Source/VolunteerReporting/Web/Models/AutomaticReplyModel.cs

```diff
@@ -0,0 +1,12 @@
+using Events.External;
+
+namespace Web.Controllers
+{
+    public class AutomaticReplyModel
```

Comment:
Does the technical aspect Model add anything as a suffix?

Reply:
See my other comment [above](#discussion-diff-151078993R7)

Reply:
oh, this was another class than the description of the enum... It's similar in this case, there will be one read model for default automatic reply and one for automatic reply. In the API, you will get a combined list, where you get a default unless it's specified. So, there is a need for wrapping these two models into one, and of course to use a naming that describes this.

Thinking about this, I'll go for AutomaticReplyForProject for the read model, and I can use AutomaticReply just for the API model